pulp-glue-ostree/pyproject.toml

```diff
@@ -23,7 +23,7 @@ classifiers = [
   "Typing :: Typed",
 ]
 dependencies = [
-  "pulp-glue>=0.20.0,<0.26",
+  "pulp-glue>=0.23.1,<0.26",
```

Comment:
This is a breaking change. Make sure you do not backport, but release a new y with this.